src/Mod/Spreadsheet/Gui/SheetModel.cpp

```diff
@@ -512,29 +533,12 @@ QVariant SheetModel::data(const QModelIndex& index, int role) const
 QVariant SheetModel::headerData(int section, Qt::Orientation orientation, int role) const
 {
     if (role == Qt::SizeHintRole) {
-        if (orientation == Qt::Horizontal) {
-            return QVariant(
-                QSize(sheet->getColumnWidth(section), PropertyRowHeights::defaultHeight));
-        }
-        else {
-            return QVariant(
-                QSize(PropertyColumnWidths::defaultHeaderWidth, sheet->getRowHeight(section)));
-        }
+        const int w = (orientation == Qt::Horizontal ? sheet->getColumnWidth(section) : PropertyColumnWidths::defaultHeaderWidth);
+        const int h = (orientation == Qt::Horizontal ? PropertyRowHeights::defaultHeight : sheet->getRowHeight(section));
+        return QSize{w, h};
```

Comment:
I'd slightly prefer here to use full names like width and height. We allow one letter variables, but mostly for math stuff where you really don't have better name than "t" etc.